src/etl/rds.py

```diff
@@ -26,31 +26,39 @@ def convert_total_seconds_to_datetime(total_seconds):
 
 class RDS:
 
-    def __init__(self):
+    def __init__(self, connect_timeout=50):
```

Comment:
Is 50 seconds essentially the "cold start" tax?

Reply:
About 60 seconds is the time it takes Aurora to scale up. Upon saying that, I think 65 might be a better number.